nh3_testcubes.py

```diff
@@ -117,135 +211,25 @@ def generate_cubes(nCubes=100, nBorder=1, noise_rms=0.1,
-                                         Voff1[i], Voff2[i], Width1[i], Width2[i],
-                                         Temp1[i], Temp2[i]]):
-                hdu11.header[kk] = vv
-        hdu11.header['TMAX'] = Tmax11
-        hdu11.header['TMAX-1'] = Tmax11a
-        hdu11.header['TMAX-2'] = Tmax11b
-        hdu11.header['RMS'] = noise_rms
-        hdu11.header['CRVAL3'] = 23694495500.0
-        hdu11.header['RESTFRQ'] = 23694495500.0
-        hdu11.writeto(output_dir + '/random_cube_NH3_11_'
-                      + '{0}'.format(i).zfill(nDigits)
-                      + '.fits',
-                      overwrite=True)
-        hdu22 = fits.PrimaryHDU(cube22)
-        for kk in hdrkwds:
-            hdu22.header[kk] = hdrkwds[kk]
-            for kk, vv in zip(truekwds, [nComps[i], logN1[i], logN2[i],
-                                         Voff1[i], Voff2[i], Width1[i], Width2[i],
-                                         Temp1[i], Temp2[i]]):
-                hdu22.header[kk] = vv
-        hdu22.header['TMAX'] = Tmax22
-        hdu22.header['TMAX-1'] = Tmax22a
-        hdu22.header['TMAX-2'] = Tmax22b
-        hdu22.header['RMS'] = noise_rms
-        hdu22.header['CRVAL3'] = 23722633600.0
-        hdu22.header['RESTFRQ'] = 23722633600.0
-        hdu22.writeto(output_dir + '/random_cube_NH3_22_'
-                      + '{0}'.format(i).zfill(nDigits) + '.fits',
-                      overwrite=True)
-
-        if ml_output:	
-            # Grab central pixel and normalize
-            loc11 = cube11[:,1,1].reshape(1000,1)
-            loc11 = loc11/np.max(loc11)
-            # Grab 3x3 average and normalize
-            glob11 = np.mean(cube11.reshape(1000,9),axis=1)
-            glob11 = glob11/np.max(glob11)
-            z = np.column_stack((loc11,glob11))
-            # Append to arrays
-            out_arr.append(z)
-            out_y.append(nComps[i])
-
-    if ml_output:
-        out_y1 = np.where(np.array(out_y)==1, 1, 0)
-        out_y2 = np.where(np.array(out_y)==0, 1, 0)
-        out_y3 = np.where(np.array(out_y)==2, 1, 0)
-        with h5py.File('nh3_three_class.h5', 'w') as hf:
-	          hf.create_dataset('data', data=np.array(out_arr))
-	          hf.close()
-        with h5py.File('labels_nh3_three_class.h5', 'w') as hf:
-	          hf.create_dataset('data', data=np.column_stack((out_y1, out_y2, out_y3)))
-	          hf.close()
+    hdu = fits.PrimaryHDU(cube)
+    for kk in hdrkwds:
+        hdu.header[kk] = hdrkwds[kk]
+        for kk, vv in zip(truekwds, [nComps, logN1, logN2,
+                                     Voff1, Voff2, Width1, Width2,
+                                     Temp1, Temp2]):
+            hdu.header[kk] = vv
+    hdu.header['TMAX'] = Tmax
+    hdu.header['TMAX-1'] = Tmax_a
+    hdu.header['TMAX-2'] = Tmax_b
+    hdu.header['RMS'] = noise_rms
+    hdu.header['CRVAL3'] = 23694495500.0
+    hdu.header['RESTFRQ'] = 23694495500.0
```

Comment:
I'm not seeing any logic that switches this to 23.722 GHz for the (2,2) line?  This seems important.